frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -52,6 +60,34 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
                 },
             },
         ],
+        person: [
+            null as PersonType | null,
+            {
+                loadPerson: async (id: string): Promise<PersonType> => {
+                    const response = await api.get(`api/person/?distinct_id=${id}`)
+                    if (!response.results.length) {
+                        router.actions.push('/404')
+                    }
+                    return response.results[0]
+                },
+                setPerson: (person: PersonType): PersonType => {
```

Comment:
This should be in the reducer as loaders should load data from API.

Reply:
That doesn't work because `person` is registered in a loader, if I registered the `person` reducer to be able to respond to a `setPerson` action, it messes with the loader (because the loader automatically creates a reducer with the same name). The alternative is manually register the actions, reducers & listeners that the loader creates, but I don't think that's worth it.